common/Makefile.am

```diff
@@ -6,6 +6,10 @@ ithemedir       = $(DESTDIR)$(themedir)
 ithemedarkerdir   = $(DESTDIR)$(themedarkerdir)
 ithemedarkdir  = $(DESTDIR)$(themedarkdir)
 
+install-exec-hook:
+	@ echo "Coping/upgrading arc-theme-upgrade script to /usr/bin/"
+	@ cp -f ../arc-theme-upgrade /usr/bin/
```

Comment:
`cp -f ../arc-theme-upgrade /usr/bin/` causes `make distcheck` to fail because the relative path. It should be `cp -f $(srcdir)/arc-theme-upgrade /usr/bin/`